markdown/1.x/en/api/ff-checkout-tracking.api.md

```diff
@@ -6,6 +6,11 @@ ___
 | **disable-auto-tracking**&nbsp;(Boolean) (default: false) | Controls whether the element sends a tracking request upon its rendering. |
 | **records**&nbsp;(Array) | The records returned from FACT-Finder. |
 
+### Methods
+| Name | Description |
+| ---- | ----------- |
+| **trackCheckoutItems** | Dispatches a tracking checkout event for each of its `ff-checkout-tracking-item` children. |
```

Comment:
I think we say `for all children` and `for each child`.

Reply:
[Resolve for the same reason as the following comment](https://github.com/FACT-Finder-Web-Components/documentation/pull/192#discussion_r296552460)